CMakeLists.txt

```diff
@@ -29,8 +29,9 @@ option(USE_PREBUILT_LIBXDIFF_PACKAGE "Use the pre-built version of the LibXDiff
 option(USE_PREBUILT_LLVMCLANG_PACKAGE "Use the pre-built version of the LLVMClang package" ON)
 option(USE_PREBUILT_OAUTH_PACKAGE "Use the pre-built version of the OAuth package" ON)
 option(USE_PREBUILT_OPENSSL_PACKAGE "Use the pre-built version of the OpenSSL package" ON)
-option(USE_PREBUILT_PYTHON_PACKAGE "Use the pre-built version of the Python package" ON)
+option(USE_PREBUILT_PYTHON_LIBRARY_PACKAGE "Use the pre-built version of the Python library package" ON)
```

Comment:
Just wondering: why `Python library` and not simply `Python`?

Reply:
The plugin was called `Python` for a long time but then I thought it made more sense to use this name for the CLI plugin that provides a Python shell prompt and runs code in the OpenCOR environment.